solarforecastarbiter/datamodel.py

```diff
@@ -117,6 +117,11 @@
     if not k.startswith('VERSION IDENTIFIER') and
     not k.startswith("RESERVED")
 ]) + tuple(DERIVED_MASKS.keys())
+ALLOWED_INTERVAL_LABELS = ('beginning', 'ending', 'instant', 'event')
+ALLOWED_INTERVAL_VALUE_TYPES = (
+    'interval_mean', 'interval_max', 'interval_min', 'interval_median',
+    'instantaneous')
+ALLOWED_AGGREGATE_TYPES = ('sum', 'mean', 'median', 'max', 'min', 'std')
```

Comment:
`std` is not included in the dashboard form. Do we need a dashboard issue for that?

Reply:
or just not include it?

Reply:
eh will keep

Reply:
https://github.com/SolarArbiter/solarforecastarbiter-dashboard/issues/377